src/cards.ts

```diff
@@ -19,11 +21,33 @@ const sets = CardInfo.reduce(
     {} as Dictionary<Expansion>
 );
 
-const cards: Card[] = CardInfo.map(card => ({
+const validCards = CardInfo.filter(card => (
+    card.set !== "Dark Ages" && 
+    card.type === "kingdom" && 
+    !(card.set === "Promo" && card.name !== "Walled Village" && card.name !== "Stash") && 
```

Comment:
please do something like `!PROMO_CARDS.includes(card.name)`

Reply:
yes but this will change.